guides/extensions/rest/setup.md

```diff
@@ -0,0 +1,31 @@
+# Setup
+
+> This guide assumes you have followed the install guides for the framework
+> (See `Getting Started` section in the sidebar) for your platform and a valid [Node.js](https://nodejs.org) project setup.
+
+## Quick Setup
+
+Using the CLI is the easiest way to get started with REST API.
+
+You can do this directly on your machine.
+
+<!--tabs-->
+
+## Directly on computer
```

Comment:
This should be h3. We can larer add the non quick setup (which is importing it as a package instead od using the cli)